var/spack/repos/builtin/packages/sqlite/package.py

```diff
@@ -14,6 +14,8 @@ class Sqlite(AutotoolsPackage):
     """
     homepage = "https://www.sqlite.org"
 
+    version('3.26.0', '9af2df1a6da5db6e2ecf3f463625f16740e036e9',
+            url='https://sqlite.org/2018/sqlite-autoconf-3260000.tar.gz')
     version('3.25.3', '00ebf97be13928941940cc71de3d67e9f852698233cd98ce2d178fd08092f3dd',
             url='https://www.sqlite.org/2018/sqlite-autoconf-3250300.tar.gz')
```

Comment:
I would remove all prior versions that contain the vulnerability. Also add a comment with the vulnerability url. See `libpng` and `singularity` for other packages where we have done this.

Reply:
I was wondering how we deal with vulnerable versions, thanks! I have pushed an updated PR.